cmd/stackit-api-manager/cmd/project.go

```diff
@@ -68,6 +69,7 @@ func init() {
 
 	// retireCmd flags
 	retireCmd.Flags().StringVarP(&identifier, "identifier", "i", "", "API Identifier")
+	retireCmd.Flags().StringVarP(&apiVersion, "api-version", "v", "", "API Version")
```

Comment:
This is redundant with the introduction of the new `retire version` target and it's not being used as far as I can see.

Reply:
Good catch, removing